litgpt/scripts/convert_hf_checkpoint.py

```diff
@@ -2,6 +2,7 @@
 
 import gc
 import json
+import os
```

Comment:
```suggestion
```

This can be removed when also applying the suggestion below.

Reply:
Done in last commit!